### ISSUE TITLE
not all long lines are reported

### ISSUE THREAD

**Issue by ToadJamb** [rrrene/credo#169]
I have the following line in a file:

```elixir
def mod_groups_for(mod, mod_list) when is_binary(mod), do: mod_groups_for([mod], mod_list)
```

I am running Credo in strict mode with line lengths set to 80. This line is 92 characters, but does not get reported. If I comment it out, it does get reported.

**Comment by rrrene** [comment]
Use `{Credo.Check.Readability.MaxLineLength, ignore_definitions: false, ignore_specs: false},` in your `.credo.exs`.

That way, definitions and `@spec` attributes will be reported as well!

**Comment by ToadJamb** [comment]
I don't have time to tinker with this now, but I will later. Those feel like they should be default settings if you have line length turned on. Or if you don't agree, then maybe set them explicitly to true in the generated `.credo.exs`. I looked at that file. If those settings had been set to true there, I would have tried switching them to false before opening an issue.
